libs/wingsdk/src/target-sim/function.ts

```diff
@@ -31,6 +32,7 @@ export class Function extends cloud.Function implements ISimulatorResource {
 
     // props.memory is unused since we are not simulating it
     this.timeout = props.timeout ?? Duration.fromMinutes(1);
+    this.concurrency = props.concurrency ?? 10;
```

Comment:
I would expect the default max concurrency to be huge, like 100 or 1000. Actually I'd expect it to be "until you run out of memory" but setting some kind of limit is probably good.

Reply:
AWS has a pretty big default limit. I think 100 should be fine for the simulator.

Reply:
Ok - updated the default to 100